src/packaging/licenses/spdx.py

```diff
@@ -0,0 +1,747 @@
+from __future__ import annotations
+
+VERSION = '3.25.0'
+
+LICENSES: dict[str, dict[str, str | bool]] = {
```

Comment:
Having this as a dict of named tuples would maybe be a notch faster (as there quite a few instances) and be nicer on typing:
```python
import typing

class License(typing.NamedTuple):
    id: str
    deprecated: bool

LICENSES: dict[str, License] = { ... }

```

Reply:
That would be significantly slower to import.

Reply:
We have a middle ground of using TypedDict…

Reply:
396e4ef69853fd176bb7b7bfc665808ab44d31f9